at_contact/pubspec.yaml

```diff
@@ -1,7 +1,7 @@
 name: at_contact
 description: A Dart library for managing contact data that developers can use for their applications.
 documentation: https://atsign.dev/docs/
-version: 2.0.2
+version: 2.0.3
```

Comment:
dependent library at_utils, at_commons you have to change to 2.0.4

Reply:
changed to recent versions